p2p/net/mock/mock_test.go

```diff
@@ -528,7 +528,7 @@ func TestLimitedStreams(t *testing.T) {
 	}
 
 	wg.Wait()
-	if !within(time.Since(before), time.Second*2, time.Second) {
+	if !within(time.Since(before), time.Second*3, time.Second) {
```

Comment:
We're now more eager to send an Identify Push: We do it even when we don't know if the peer support push yet (i.e. before the initial identify exchange completes). This means that we now have to fit the Identify message through the (bandwidth-limited) pipe as well, which messes with the timing here.